README.md

```diff
@@ -59,4 +59,5 @@ Read [How to contribute](https://github.com/Aguafrommars/TheIdServer/tree/master
 
 ## OIDC Certification test result
 
-https://www.certification.openid.net/plan-detail.html?plan=ZKco5LJhicIlT
\ No newline at end of file
+The server pass the [oidcc-basic-certification-test-plan](
```

Comment:
You might want to change this as follows:

The server passes the [oidcc-basic-certification-test-plan](https://www.certification.openid.net/plan-detail.html?plan=ZKco5LJhicIlT&public=true) with some warnings. It is anticipated that it will pass the certification process, but we need your assistance.  Please sponsor this project to help us pay the required certification fee.

Reply:
I'll add a link to certification fee page : https://openid.net/certification/fees/

Reply:
You tell the reader that you need sponsorship and provide a link to explain the cost.  Consider adding a direct link to https://liberapay.com/aguacongas to allow the reader to make a financial contribution for convenience.

Reply:
Not sure I prefer liberapay but I can add 3 links like that : 

Choose your favorite:

* [github sponsor](https://github.com/sponsors/aguacongas),
* [liberapay](https://liberapay.com/aguacongas)
* [issuehunts](https://issuehunt.io/r/Aguafrommars/TheIdServer)